### ISSUE TITLE
[css-egg] Note about astronomical units doesn't match physical reality

### ISSUE THREAD

**Issue by SelenIT** [w3c/csswg-drafts#2491]
In [the 'Astronomical units' section](https://drafts.csswg.org/css-egg-1/#astro-units), the parsec unit (`pc`) is defined as the absolute unit equal to 3.086E18 `cm`, and there is the following note:
> This specification does not address relativistic effects of scrolling or animations and transitions at (or beyond) the speed of light.

However, all browsers I tested don't preserve this proportion. Instead, they render 1 parsec as something about 0.38 to 0.4 `cm` (see JSfiddle example: https://jsfiddle.net/oxqoqkxh/1/).

The only explanation to this phenomenon I can come up with is that, according to the Special Relativity theory, "traditional" units and astronomical units are measured in the different reference frames: the Lorentz factor of the reference frame where "traditional" absolute units are measured is extremely lower than that of the reference frame used for astronomical units. While the Lorentz factor of the former is usually below 2 (e.g., on iPhone and iPad mini `1in` at 100% scale is about 0.59 physical inches), the latter is several orders of magnitude higher. And because the proportion of the "static" centimeter and the "Lorentz transformed" parsec is nearly the same in all browsers, I suppose that the speed rate of these two reference frames must be a fundamental constant.

I haven't calculated which speed does the Lorentz factor used for astronomical units correspond, but given that the CSS EGG spec has many references to CERN, I guess that this speed has something to do with the LHC (maybe it's the speed of the proton beam at its maximum energy?).

Technically, this speed is not "the speed of light or beyond", so I can't say that the current Note in the spec is _absolutely_ wrong. But because it's 1) very close to it and 2) its Lorentz factor is _in constant proportion_ to that of other physical units in all browsers, I believe this fact is also worth noting in the spec!

**Comment by Loirooriol** [comment]
Sadly the parsec unit `pc` conflicts with the picas unit. Anyways I think one parsec is too small for some usecases, e.g. to represent the Milky Way at real scale. So I propose replacing the parsec unit with `kpc`, i.e. kiloparsec.

**Comment by jonjohnjohnson** [comment]
I just hope this ships before I finish the Kessel Run app I'm building.

**Comment by SelenIT** [comment]
Since the picas unit doesn't seem used much in the wild, maybe this conflict can be resolved by replacing it with _picopicas_ (1e-9 picas) unit? I assume that, given the constantly growing screen resolutions of the mobile devices, this unit could be really useful in VR/AR applications like _PokemonGo_ in the super-high-definition Web context...

**Comment by fuchsia** [comment]
I did calculate the speed, and inertial frames ain't gonna cut it: the faster frame would have to be moving at something like `0.9999999999999999999999999999999999999905c`. And javascript says that number—approximately `1 - 9.4E-39`—is equal to the speed of light. _(Experts: I had to Taylor-expand the square root. So I only used one term, effectively discarding terms of O(distance_ratio**4) and above.)_

No, I think we can only reasonably explain the discrepancy via gravitational contraction. Specifically, CSS is a massive black hole and we're all deep within that region where events can be captured but not bubble up back to the initiating universe.

All your Christoffel symbols are belong to CSSWG.
